app/managers.py

```diff
@@ -0,0 +1,45 @@
+import sqlite3
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("../cinema.sqlite")
+        self.table_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self.table_name} "
+            f"(first_name, last_name) VALUES (?, ?) ",
+            (first_name, last_name)
+        )
+        self._connection.commit()
+
+    def all(self) -> list[Actor]:
+        actors_cursor = self._connection.execute(
+            f"SELECT * FROM {self.table_name}"
+        )
+        return [
+            Actor(*row) for row in actors_cursor
+        ]
+
+    def update(
+            self,
+            id_to_update: int,
```

Comment:
There is no need to explicitly indicate an operation in the parameter name, `id` would be enough here and in the other similar places
```suggestion
            id: int,
```